NYPLAudiobookToolkit/Utils/MediaProcessor.swift

```diff
@@ -300,4 +285,24 @@ class MediaProcessor {
             }
         }
     }
+    
+    private static func seek(filehandle: FileHandle, offset: UInt64) throws {
+        filehandle.seek(toFileOffset: offset)
```

Comment:
I didn't see a crash for `seek(toOffset:)` but I also don't think I got far enough in execution to reach this point.  Just wondering if you did and if this is actually needed.

Reply:
I did not see this but it is a part of the FileHandle API changes so I didnt want to risk it